### PR TITLE
update codeowners for aws bedrock data streams

### PR DESCRIPTION
**Pull request by tommyers-elastic**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -71,7 +71,9 @@
 /packages/aws/data_stream/waf @elastic/security-service-integrations
 /packages/aws/kibana @elastic/obs-ds-hosted-services @elastic/obs-infraobs-integrations
 /packages/aws/manifest.yml @elastic/obs-ds-hosted-services @elastic/security-service-integrations @elastic/obs-infraobs-integrations
```
Comment (tommyers-elastic) [inline .github/CODEOWNERS:73]: do i need to add a line for the manifest too? i only see this on aws and system integration, and i'm not sure why

Comment (lalit-satapathy) [reply .github/CODEOWNERS:73]: In which case all PRs will be routed to both teams.

**Review by lalit-satapathy — APPROVED** [review]
LGTM.

File: .github/CODEOWNERS (hunk 1 above)
Comment (tommyers-elastic) [reply .github/CODEOWNERS:73]: isn't this handled by having both teams on the root package path?

Comment (lalit-satapathy) [reply .github/CODEOWNERS:73]: yes, thats my understanding.

Comment (lalit-satapathy) [reply .github/CODEOWNERS:73]: There may be some redundant entries there.

**Pull request merged**